web/skins/classic/css/dark/skin.css

```diff
@@ -42,6 +42,10 @@ h4 {
     color: #117AAd;
 }
 
+a {
```

Comment:
This shouldn't be neccessary as you've added it to the base... skin.css.. or base/skin.css.  We always include base.

Reply:
Of course you're right, sorry.
I changed code.